英初稿The (Second) Deadliest Virus-360p.srt

```diff
@@ -1,664 +1,762 @@
-﻿1
+1
 00:00:00,830 --> 00:00:05,700
-Few of the monsters that evolution created have been so successful at hurting us
+進化が生んだ怪物たち
+なかでも屈指の被害を生んだのが
 
 2
 00:00:05,700 --> 00:00:09,420
-as the variola virus, responsible for smallpox
+ヴァリオラウイルス
+天然痘の病原だ
 
 3
 00:00:09,420 --> 00:00:12,770
-The carnage it caused was so terrible and merciless 
+この悲惨で無慈悲な虐殺により
 
 4
 00:00:12,770 --> 00:00:17,460
-that it compelled humankind, for the first time,  to act truly globally.
+人類は初めて
+真にグローバルな行動を求められた
 
 5
 00:00:17,460 --> 00:00:21,510
-It was one of the   greatest wins of our species over the ancient  powers of nature,
+自然の脅威に対する
+人類最大の勝利の一つで
 
 6
 00:00:21,510 --> 00:00:23,230
-all made possible by… 
+その立役者は…
 
 7
 00:00:23,230 --> 00:00:24,750
-cows
+牛たちだった
 
 8
 00:00:33,080 --> 00:00:37,810
-Variola is a virus, a tiny machine that only  seeks to reproduce itself.
+ヴァリオラはウイルスで
+自己増殖を追求する小型マシンだ
 
 9
 00:00:37,810 --> 00:00:43,690
-Evidence of It has been found in Egyptian mummies and in writing from India and China as old as 3000 years.
+エジプトのミイラにも
+3000年前のインドや中国の文書にも痕跡がある
 
 10
 00:00:43,690 --> 00:00:48,960
-1300 years ago smallpox killed up to a third of  Japan’s population.
+1300年前
+天然痘は日本の人口の1/3を殺し
 
 11
 00:00:48,960 --> 00:00:53,260
-By the sixteen hundreds, it   was one of the major causes of death worldwide.
+1600年ごろには
+世界の主な死因の一つとなっていた
 
 12
 00:00:53,260 --> 00:00:57,140
-In late 18th century Europe, it killed 400,000 a year.
+18世紀後半のヨーロッパでは
+年に40万人が死に
 
 13
 00:00:57,140 --> 00:01:00,960
-Every third person who went blind did so because of this virus.
+失明の1/3は
+このウイルスが原因だった
 
 14
 00:01:00,960 --> 00:01:04,290
-Even in the 20th century, a hot second ago in history,
+つい最近の20世紀でも
 
 15
 00:01:04,290 --> 00:01:07,000
-it still killed at  least 300 million people.
+少なくとも3億人が死んでいる
 
 16
 00:01:07,000 --> 00:01:11,710
-Smallpox is an abusive monster that returns over and over and over again, 
+天然痘は執拗な怪物で
+何度も何度も再発しては
 
 17
 00:01:11,710 --> 00:01:15,690
-killing, maiming, and disrupting societies.
+殺し 障害を残し
+社会を崩壊させる
 
 18
 00:01:15,690 --> 00:01:19,070
-How could variola be so incredibly deadly for so long
+なぜこれほど長く猛威をふるったのか
 
 19
 00:01:19,070 --> 00:01:22,350
-and how could we have forgotten its horror so quickly?
+その恐怖がすぐに風化した理由は？
 
 20
 00:01:22,350 --> 00:01:28,040
-In 2023, there are only two laboratories left where the living virus is officially stored for research:
+2023年現在 生きたヴァリオラを
+公式に保管している研究所は二か所で
 
 21
 00:01:28,040 --> 00:01:31,480
-in Koltsovo, Russia and in Atlanta, USA
+ロシアのコルツォボ
+アメリカのアトランタだ
 
 22
 00:01:31,480 --> 00:01:33,110
-Which is certainly a good idea 
+実際 いいやり方だ
 
 23
 00:01:33,110 --> 00:01:35,710
-what could possibly go wrong?
+何か問題でも？

```

Comment:
見直して気づいたのですが、22と23の間に原文の `because` が抜け落ちてしまってますね。それを踏まえて提案です。
`what could possibly go wrong?` を反語として解釈して

`実際　いいやり方で`
`大事故を防げる`

なんていかがでしょうか。
意訳過ぎる気もしますが…